### PR TITLE
[WIP] initial commit for german website

### PR DESCRIPTION
**Pull request by dkhgh**
this is done, can be tested (build and run a simple server, don't test using `middleman server`)

TO DO

* [x] use german reference stories from data
* [ ] ~~show german featured blog on german site~~ (see https://github.com/DefactoSoftware/website/pull/87)

### CODE REVIEW

**Comment by dkhgh** [comment]
this is done, can be tested (build and run a simple server, don't test using `middleman server`)

TO DO

* [ ] use german reference stories from data
* [ ] show german featured blog on german site

File: source/localizable/_header.erb [hunk 1]
```diff
@@ -1,22 +1,33 @@
...
+        <li><%= nav_link_to t("header.home"), "/" %></li>
+        <li><%= nav_link_to t("header.products"), "/producten" %></li>
+        <li><%= nav_link_to t("header.services"), "/diensten" %></li>
+        <li><%= nav_link_to t("header.about"), "/over-ons" %></li>
+        <li><%= nav_link_to t("header.blog"), "/blog" %></li>
+        <li><%= nav_link_to t("header.contact"), "/contact" %></li>
         <li class="flags">
-          <%= link_to image_tag("flags/de.gif", alt: "Deutsch"), "http://de.defacto.nl", target: "_blank" %>
+        <% if I18n.locale != :nl %>
+          <% link_to current_page.url.sub! '/de', '' do %>
```
Comment (fatboypunk) [inline source/localizable/_header.erb]:
```ruby
link_to current_page.url.sub! '/de/', '/' do
```

**Comment by sn3p** [comment]
- [ ] translate links in footer

**Comment by sn3p** [comment]
Localizing the Blog is a pain in the ass, but I'm making some progress.
Still need to figure some things out, so I'm moving this to a separate PR (https://github.com/DefactoSoftware/website/pull/87).

**Comment by dkhgh** [comment]
what about the rest of the to-do's in this PR?

**Comment by dkhgh** [comment]
to do: test titles + forms, push to divshot, change redirects, test again, GO!

**Comment by dkhgh** [comment]
@sn3p can we hide the blog + links to it on the german site for now?

**Pull request merged**